### PR TITLE
[doc] add multi-gpu node classification tutorial

### PR DESCRIPTION
**Pull request by Rhett-Ying**
## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] I've leverage the [tools](https://docs.google.com/document/d/1iHyj7zlmygKSk5gBPsqIqL5ASPzJSPREaNT_QdsiYA4/edit) to beautify the python and c++ code.
- [ ] The PR is complete and small, read the [Google eng practice (CL equals to PR)](https://google.github.io/eng-practices/review/developer/small-cls.html) to understand more about small PR. In DGL, we consider PRs with less than 200 lines of core code change are small (example, test and documentation could be exempted).
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

File: tutorials/multi/2_node_classification.py [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+        )
+        / world_size
+    )
+    dist.reduce(tensor=test_acc, dst=0)
+    dist.barrier()
+    if rank == 0:
+        print(f"Test Accuracy {test_acc.item():.4f}")
+
+
+######################################################################
```
[the remark this answers is not in the document]
Comment (Rhett-Ying) [reply tutorials/multi/2_node_classification.py:357]: not runnable during doc build. `mp.spawn` crashed.

[the remark this answers is not in the document]
Comment (Rhett-Ying) [reply tutorials/multi/2_node_classification.py:357]: exit if no cuda is available

File: tutorials/multi/2_node_classification.py [hunk 2]
```diff
@@ -0,0 +1,396 @@
+"""
+Single Machine Multi-GPU Minibatch Node Classification
+======================================================
+
+In this tutorial, you will learn how to use multiple GPUs in training a
```
Comment (frozenbugs) [inline tutorials/multi/2_node_classification.py:5]: nit: better to make full use of 80 chars before change line.

**Review by frozenbugs — APPROVED** [review]

File: tutorials/multi/2_node_classification.py (hunk 2 above)
Comment (Rhett-Ying) [reply tutorials/multi/2_node_classification.py:5]: ok

**Pull request merged**